internal/explorer/db/postgres.go

```diff
@@ -464,6 +464,9 @@ func (d *PostgresDatabase) GetNodes(filter types.NodeFilter, limit types.Limit)
 		q = q.Limit(int(limit.Size)).
 			Offset(int(rand.Intn(int(count)) - int(limit.Size)))
 	} else {
+		if filter.AvailableFor != nil {
+			q = q.Order("(case when farm.dedicated_farm then 1 else 2 end)")
```

Comment:
the dedicated nodes can be a node in a dedicated farm or a node with no contracts. can't we depend on the rent contract of the node? if it has a positive number, it means dedicated and rented so it can go to the top of the list

Reply:
Not sure what is the value we are depending on in the contract. Twin id?

Reply:
Also another thought, since `dedicated` property is the `farm.dedicated_farm` does that mean that we can't depend on that value to know if a node is dedicated or not?